exits.go

```diff
@@ -0,0 +1,79 @@
+package main
+
+import (
+	"math/rand"
+	"time"
+	"fmt"
+)
+
+func printMatrix(matrix [][]int){
+	for row := range matrix {
+		for col := range matrix[row] {
+			fmt.Printf("%d ", matrix[row][col])
+		}
+		fmt.Printf("\n")
+	}
+	fmt.Printf("\n")
+}
+
+func generateExits(floor [][]int){
+	sideExits := make([]int, 4)
+	lenF := len(floor)
+	rand.Seed(time.Now().UnixNano())
+	nExits := rand.Intn(5) + 1
+	for i:=0;i<nExits;{
+		side := rand.Intn(4)
+		indexExit := rand.Intn(lenF)
+		if (sideExits[side]>1){
+			continue
+		}
+		switch side {

```

Comment:
Could the switch case be in a separate function? so the generateExits function is not so long and is a little more clear?